usaspending_api/awards/tests/unit/test_location_filter_geocode.py

```diff
@@ -83,7 +83,7 @@ def award_data_fixture(db):
 
 def test_geocode_filter_locations(award_data_fixture):
 
-    to = ContractAwardSearchMatview.objects
+    to = AwardSearch.objects
```

Comment:
Might be worth filtering this down to only contract awards that way it matches previous expected functionality for the test. This is currently passing most likely due to limitations on the test data created.

Not worth blocking over, but would help to make the test a bit more resilient.